python/python-core/tests/test_vehicle_routing.py

```diff
@@ -39,7 +40,11 @@ class Visit:
                                       target_method_name='update_arrival_time')] = field(default=None)
     piggyback_arrival_time: Annotated[
         Optional[datetime],
-        PiggybackShadowVariable(shadow_variable_name='arrival_time')] = field(default=None)
+        PiggybackShadowVariable(shadow_variable_name='arrival_time', shadow_entity_class=Visit)] = field(default=None)
```

Comment:
Visit is not defined yet (this is inside the Visit class); this code would not run/crash at runtime. Remove the `shadow_entity_class` here.

Reply:
Will the import `from __future__ import annotations` fix the issue related to the import order?

Reply:
`from __future__ import annotations` might be deprecated in the near future; see https://peps.python.org/pep-0749/ ; I say just remove it, since there is a test for it elsewhere.